ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,28 @@
+Funding
+=======
+The following institutions, missions, and programs have provided funding
+for pysatSpaceWeather development.
+
+Institutions
+------------
+ - Cosmic Studio
```

Comment:
@JonathonMSmith shows up on the contributor's list, so we may need CUA here too.